modules/apps/27-interchain-accounts/module.go

```diff
@@ -136,13 +140,19 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	}
 
 	if am.hostKeeper != nil {
+		hosttypes.RegisterMsgServer(cfg.MsgServer(), hostkeeper.NewMsgServerImpl(am.hostKeeper))
 		hosttypes.RegisterQueryServer(cfg.QueryServer(), am.hostKeeper)
 	}
 
 	m := controllerkeeper.NewMigrator(am.controllerKeeper)
 	if err := cfg.RegisterMigration(types.ModuleName, 1, m.AssertChannelCapabilityMigrations); err != nil {
 		panic(fmt.Sprintf("failed to migrate interchainaccounts app from version 1 to 2: %v", err))
 	}
+
+	hostm := hostkeeper.NewMigrator(am.hostKeeper)
+	if err := cfg.RegisterMigration(types.ModuleName, 2, hostm.MigrateParams); err != nil {
+		panic(fmt.Sprintf("failed to migrate interchainaccounts host params from version 2 to 3: %v", err))
```

Comment:
Since we will probably migrate together both controller and host params, maybe we can do something here to bump the version only once. But maybe we can deal with that in a later PR after merging also the migration of the controller params.

```suggestion
		panic(fmt.Sprintf("failed to migrate interchainaccounts app from version 2 to 3: %v", err))
```

Reply:
done, decided to increase the version now, I assumed the two would be done together but this is safer

Reply:
Yep, the migration handler can be configured like this with an anonymous function

```go
if err := cfg.RegisterMigration(types.ModuleName, 2, func(ctx sdk.Context) error {
    if err := hostMigrator.MigrateParams(ctx); err != nil {
        return err
    }

    if err := controllerMigrator.MigrateParams(ctx); err != nil {
        return err
    }
}); err != nil {
	panic(fmt.Sprintf("failed to migrate interchainaccounts app from version 2 to 3: %v", err))
}
```

Reply:
this can be done later, lgtm